content/docs/tasks/security/role-based-access-control/index.md

```diff
@@ -66,6 +66,124 @@ for the list of supported keys in `constraints` and `properties`.
     If you refresh the page several times, you should see different versions of reviews shown in the product page,
     presented in a round robin style (red stars, black stars, no stars)
 
+## RBAC policy permissive mode
+
+RBAC permissive mode is for testing a new RBAC policy works as expected before
+rolling out to production. This sections shows how RBAC permissive mode is used.
```

Comment:
I think it would be nicer to explain a little bit how permissive mode work and why it can be used to safely test a RBAC policy before rolling out to production.

You don't need to go too deep here, you can put more explanation under the Authorization concept page.

Reply:
concept probably would be a better place, will do in separated PR.